Tests/UtilsTests/FindUserTest.cs

```diff
@@ -0,0 +1,32 @@
+using Xunit;
+
+namespace BackEndFucntions.Tests.UtilsTests
+{
+    public static class FindUserTest
+    {
+        [Fact]
+        public static void DatabaseNotFound()
+        {
+            const string ID = "0";
```

Comment:
Can ID to USER_KEY since it is using userkey is the parameter for getting the userID